src/main/java/com/hedvig/contract/shared/switchers/SwitchableSwitcherCompany.kt

```diff
@@ -4,14 +4,30 @@ enum class SwitchableSwitcherCompany(val realName: String, val email: String) {
   FOLKSAM("Folksam", "annullationer-sak@folksam.se"),
   TRYGG_HANSA("Trygg-Hansa", "annullation-privat@trygghansa.se"),
   ICA("ICA Försäkringar", "forsakring@ica.se"),
-  TRE_KRONOR("Tre Kronor", "support@3kronor.se");
+  TRE_KRONOR("Tre Kronor", "support@3kronor.se"),
+  IF_NO("If", "oppsigelser@if.no"),
+  GJENSIDIGE("Gjensidige", "oppsigelse@gjensidige.no"),
+  TRYG("Tryg", "oppsigelser.privat@tryg.no"),
+  FREMTIND("Fremtind / Spare Bank 1 / DNB", "forsikring.privat@fremtind.no"),
+  EIKA("Eika", "oppsigelser@eika.no"),
+  FRENDE("Frende", "oppsigelse@frende.no"),
+  STOREBRAND("Storebrand", "oppsigelser@storebrand.no"),
+  CODAN("Codan", "oppsigelse@codanforsikring.no");
 
   companion object {
-    fun from(switchedFrom: String?): SwitchableSwitcherCompany? = when(switchedFrom?.toLowerCase()) {
+    fun from(switchedFrom: String?): SwitchableSwitcherCompany? = when (switchedFrom?.toLowerCase()) {
       "folksam" -> FOLKSAM
       "trygg-hansa", "trygghansa", "trygg hansa", "trygg_hansa" -> TRYGG_HANSA
       "ica", "ica forsakringar", "ica_forsakringar", "ica-forsakringar", "ica försäkringar", "ica försäkring" -> ICA
       "tre kronor", "tre-kronor", "tre_kronor" -> TRE_KRONOR
+      "if no" -> IF_NO
```

Comment:
Maybe this might happen?
```suggestion
      "if no", "if_no" -> IF_NO
```

Reply:
true will push to master :cowboy_hat_face: